src/callbacks/onPreprocess/flattenData/calculateRatios.js

```diff
@@ -0,0 +1,25 @@
+export default function calculateRatios(d, participant_obj) {
```

Comment:
Wonder if we're killing a fly with a sledgehammer here. I think we could just do a simple mapping for the raw values and then `participant_obj.rRatio = ALT/ALP` and call it a day. Don't think the other ratios are needed at this point.